data/pigui/libs/equipment-outfitter.lua

```diff
@@ -0,0 +1,648 @@
+			end)
+		end)
+	end
+
+	ui.separator()
+	ui.spacing()
+
+	if self.selectedEquip then
+		ui.textWrapped(self.selectedEquip.equip:GetDescription())
+	elseif self.currentEquip then
+		ui.textWrapped(self.currentEquip.equip:GetDescription())
+	end
+
+	ui.spacing()
+
+	if self.compare_stats then
+
+		if ui.beginTable("##CompareEquipStats", 3) then
+
+			ui.tableSetupColumn("##name", { "WidthStretch" })
+			ui.tableSetupColumn("##selected", { "WidthFixed" })
+			ui.tableSetupColumn("##current", { "WidthFixed" })
+
+			for _, row in ipairs(self.compare_stats) do
+				ui.tableNextRow()
+				self:renderCompareRow(row.label, row.a, row.b)
+			end
+
+			ui.endTable()
+
+		end
+
+	end
+
+end
+
+function Outfitter:render()
+
+	local spacing = ui.getItemSpacing()
+	local panelWidth = (ui.getContentRegion().x - spacing.x * 4.0) / 2
+
+	ui.child("##ListPane", Vector2(panelWidth, 0), function()
+
+		ui.withStyleVars({ FrameRounding = 4 }, function()
+			ui.withFont(orbiteer.heading, function()
+				if ui.iconButton("back", icons.decrease_thick, l.CLOSE, nil, nil, styles.IconButtonPadding * 1.5) then
+					self:message("onClose")
+				end
+
+				ui.sameLine()
+
+				ui.alignTextToFramePadding()
+				ui.text(self.replaceEquip and l.REPLACE_EQUIPMENT or l.AVAILABLE_FOR_PURCHASE)
+			end)
+
+			ui.withFont(pionillium.details, function()
+				self:drawSortRow()
+			end)
+		end)
+
+
+		ui.spacing()
+
+		local buttonLineHeight = 0.0
+		local spacing = ui.getItemSpacing()
+
+		if self.selectedEquip then
+			buttonLineHeight = buttonLineHeight + getCustomButtonHeight() + spacing.y
+		end
+
+		if self.currentEquip then
+			buttonLineHeight = buttonLineHeight + getCustomButtonHeight() + spacing.y
+		end
+
+		if buttonLineHeight > 0.0 then
+			buttonLineHeight = buttonLineHeight + spacing.y * 2.0
+		end
+
+		ui.child("##EquipmentList", Vector2(0, -buttonLineHeight), function()
+			for _, data in ipairs(self.equipmentList) do
+				local clicked = self:drawEquipmentItem(data, data == self.selectedEquip)
+
+				if clicked then
+					self:message("onSelectItem", data)
+				end
+
+				local doubleClicked = clicked and ui.isMouseDoubleClicked(0)
+
+				if doubleClicked and data.available then
+					self:message("onBuyItem", data.equip)
+				end
+			end
+		end)
+
+		if self.selectedEquip or self.currentEquip then
+			ui.separator()
+			ui.spacing()
+		end
+
+		if self.selectedEquip then
+			self:drawBuyButton(self.selectedEquip)
+		end
+
+		if self.currentEquip then
+			self:drawSellButton(self.currentEquip)
+		end
+
+	end)
+
+	ui.sameLine(0, spacing.x * 4)
+
+	local linePos = ui.getCursorScreenPos() - Vector2(spacing.x * 2, 0)
+	ui.addLine(linePos, linePos + Vector2(0, ui.getContentRegion().y), ui.theme.styleColors.gray_500, 2.0)
+
+	ui.group(function()
+		local scannerSize = Vector2(panelWidth, (ui.getContentRegion().y - ui.getItemSpacing().y) / 2.0)
+
+		ui.child("##EquipmentDetails", Vector2(panelWidth, scannerSize.y), function()
```

Comment:
TODO: The station's stock count for this equipment should be rendered somewhere here.

Reply:
Yep. Deferring this one to Stage 2. Please feel free to open an issue for this right away so it can be tracked.

Reply:
#5953 

\closing